js/aax.js

```diff
@@ -599,69 +599,94 @@ module.exports = class aax extends Exchange {
-                'info': currency,
-                'active': active,
-                'deposit': deposit,
-                'withdraw': withdraw,
-                'fee': fee,
-                'network': network,
-                'limits': {
-                    'deposit': {
-                        'min': this.safeNumber (currency, 'depositMin'),
-                        'max': undefined,
-                    },
-                    'withdraw': {
-                        'min': this.safeNumber (currency, 'withdrawMin'),
-                        'max': undefined,
+            const resultItem = this.safeValue (result, code);
+            const fee = this.safeNumber (currency, 'withdrawFee');
+            const precision = this.safeString (currency, 'withdrawPrecision');
+            const depositMin = this.safeString (currency, 'depositMin');
+            const withdrawMin = this.safeString (currency, 'withdrawMin');
+            if (resultItem !== undefined) {
+                if (resultItem['fee'] !== undefined) {
+                    const previousNetwork = this.safeString (resultItem['networks'], 0);
+                    resultItem['fees'][previousNetwork] = resultItem['fee'];
+                    resultItem['fee'] = undefined;
+                }
+                resultItem['fees'][network] = fee;
+                resultItem['networks'].push (network);
+                const previousPrecision = resultItem['precision'].toString ();
+                const previousDepositMin = resultItem['limits']['deposit']['min'].toString ();
+                const previousWithdrawMin = resultItem['limits']['withdraw']['min'].toString ();
+                resultItem['precision'] = Precise.stringMax (previousPrecision, precision);
+                resultItem['limits']['deposit']['min'] = Precise.stringMin (previousDepositMin, depositMin);
+                resultItem['limits']['withdraw']['min'] = Precise.stringMin (previousWithdrawMin, withdrawMin);
+            } else {
+                const name = this.safeString (currency, 'displayName');
+                const enableWithdraw = this.safeValue (currency, 'enableWithdraw');
+                const enableDeposit = this.safeValue (currency, 'enableDeposit');
+                const visible = this.safeValue (currency, 'visible');
+                const active = (enableWithdraw && enableDeposit && visible);
+                const deposit = (enableDeposit && visible);
+                const withdraw = (enableWithdraw && visible);
+                result[code] = {
+                    'info': currency,
+                    'id': id,
+                    'name': name,
+                    'code': code,
+                    'precision': this.parseNumber (precision),
+                    'active': active,
+                    'deposit': deposit,
+                    'withdraw': withdraw,
+                    'fee': fee,
+                    'fees': {},
+                    'networks': [ network ],
```

Comment:
the `networks` objects should be a list of network-objects with the proper structure, not only their names